### DISCUSSION TITLE
Add a server node to the teleport cluster under the condition that the node server and teleport cluster cannot ping each other

### DISCUSSION THREAD

**Discussion by longansv** [gravitational/teleport#39321] [Q&A]
I have an ssh node that cannot ping the teleport cluster so I want to configure teleport.yaml on that node to enable reverse tunnels to the teleport cluster. Can someone help me what should I edit in the node's teleport.yaml file to be able to add the node to the teleport cluster?
This is the current teleport.yaml file on node:
`version: v3
teleport:
  nodename: jumpserver-vas01
  data_dir: /var/lib/teleport
  join_params:
    token_name: /tmp/token.file
    method: token
  proxy_server: longhd98.click:443
  log:
    output: stderr
    severity: INFO
    format:
      output: text
  ca_pin: ""
  diag_addr: ""
auth_service:
  enabled: "no"
ssh_service:
  enabled: "yes"
  commands:
  - name: hostname
    command: [hostname]
    period: 1m0s
proxy_service:
  enabled: "no"
  https_keypairs: []
  https_keypairs_reload`

**Comment by webvictim** [comment]
When you say "the node cannot ping the Teleport cluster" it implies that your node does not actually have any way to connect to the Teleport cluster - is that the case? If so, Teleport unfortunately cannot solve your problem here as you need _some_ connectivity between the two.

If the node is capable of making an outbound connection to `longhd.click:443` then it should be possible to make Teleport work. Your Teleport cluster is generally expected to have an internet-routable public IP address in this instance to facilitate the connection.

**Reply by longansv** [reply]
yes, thank you for your response, I understand
